ceph-dev-new-trigger/config/definitions/ceph-dev-new-trigger.yml

```diff
@@ -177,7 +177,7 @@
                   predefined-parameters: |
                     BRANCH=${GIT_BRANCH}
                     FORCE=True
-                    DISTROS=focal bionic centos8 leap15
```

Comment:
this also matches with "luminous\|mimic\|nautilus\|octopus\" in addition to pacific. are you sure we don't need to build these branches for leap?

Reply:
@tchaikov according to condition it matches master and anything else but not `luminous|mimic|nautilus|octopus|pacific`.
```
          condition-command: |
            echo "${GIT_BRANCH}" | grep -v '\(luminous\|mimic\|nautilus\|octopus\|pacific\)'
```

Reply:
ahh, right! I missed `-v`.